geode-assembly/src/acceptanceTest/java/org/apache/geode/cache/wan/SeveralGatewayReceiversWithSamePortAndHostnameForSendersTest.java

```diff
@@ -101,8 +100,8 @@
   @BeforeClass
   public static void beforeClass() throws Exception {
     // Start locator
-    docker.execForService("locator", "gfsh", "run",
-        "--file=/geode/scripts/geode-starter-locator.gfsh");
+    docker.execForService("locator", "gfsh", "-e",
+        startLocatorCommand());
     // Start server1
     docker.execForService("server1", "gfsh", "run",
         "--file=/geode/scripts/geode-starter-server1.gfsh");
```

Comment:
I suggest we get rid of the .gfsh file here and inline the command and the same for the next two gfsh invocations.

Reply:
Good idea! I turned all of the scripts into `gfsh -e` commands inline in the java code. Thanks!